packages/kbn-rule-data-utils/src/alerts_as_data_rbac.ts

```diff
@@ -38,3 +40,28 @@ export type ValidFeatureId = keyof typeof mapConsumerToIndexName;
 export const validFeatureIds = Object.keys(mapConsumerToIndexName);
 export const isValidFeatureId = (a: unknown): a is ValidFeatureId =>
   typeof a === 'string' && validFeatureIds.includes(a);
+
+/**
+ * Prevent javascript from returning Number.MAX_SAFE_INTEGER when Elasticsearch expects
+ * Java's Long.MAX_VALUE. This happens when sorting fields by date which are
+ * unmapped in the provided index
+ *
+ * Ref: https://github.com/elastic/elasticsearch/issues/28806#issuecomment-369303620
+ *
+ * return stringified Long.MAX_VALUE if we receive Number.MAX_SAFE_INTEGER
+ * @param sortIds estypes.SearchSortResults | undefined
+ * @returns SortResults
+ */
+export const getSafeSortIds = (sortIds: estypes.SearchSortResults | undefined) => {
+  if (sortIds == null) {
+    return sortIds;
```

Comment:
This might be nitty gritty, but technically, `== null` means it can be either `null` or `undefined`. I know above it's typed to ` | undefined` but might be better to just return `undefined` explicitly just in case?

Reply:
Or type `sortIds` to be `| undefined | null`

Reply:
Agreed! will update.

Reply:
fixed: dcea9504cf03d767eae3f8a778e2d85a759b64a5